ci/README.md

```diff
@@ -0,0 +1,63 @@
+# CodeBuild CI
```

Comment:
should we move this to `docs/internal`?

Reply:
Yeah I think that makes sense. Will update the PR.